sdk/state/close.go

```diff
@@ -18,8 +18,16 @@ import (
 // 6. If A or B declines or is not responsive at any step, A or B may submit the
 //    original close tx after the observation period.
 
-func (c *Channel) closeTxs(oad OpenAgreementDetails, d CloseAgreementDetails) (txDecl *txnbuild.Transaction, txClose *txnbuild.Transaction, err error) {
-	txClose, err = txbuild.Close(txbuild.CloseParams{
+func (c *Channel) closeTxs(oad OpenAgreementDetails, d CloseAgreementDetails) (txs CloseAgreementTransactions, err error) {
```

Comment:
💡 can we add a method comment here describing how this method is essentially loading the "cached" transaction if the input open agreement and close agreement are the same as the channel ones? I think that's something not obvious but worth pointing out, and is how we call up the saved `channel.<>AgreementTransactions`

Reply:
Great point. 8709d01.